### PR TITLE
Having About page scale on larger screen sizes

### PR DESCRIPTION
**Pull request by krishnakpandian**
Problem
=======
C2D-110 [Jira](https://cruzhacks-dev.atlassian.net/browse/C2D-110)

Solution
========
What I/we did to solve this problem
* Set a max-width to prevent stretching on widescreen
* Adjusted font-responsiveness to be smaller on desktop

Change Summary:
---------------
* Adjusted stylesheets AboutBox.scss and index.scss

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------
* Insert any notes/issues, dependencies added or removed, or images

### CODE REVIEW

File: src/views/Home/components/About/AboutBox/AboutBox.scss [hunk 1]
```diff
@@ -4,7 +4,8 @@
 .about-box {
     z-index: 1;
     border-radius: 33px;
-    width: 80%;
+    width: 70%;
+    max-width: 1500px;
     height: 100%;
     background-color: var(--color-dark-navy-blue);
     margin: 50px 0 50px 0;
```
Comment (w-gao) [inline src/views/Home/components/About/AboutBox/AboutBox.scss:11]: Could we also set a min-height to `&__title` below? For example, something like:

```css
        display: flex;
        flex-direction: column;
        justify-content: center;
        min-height: 50px;
```

**Review by w-gao — APPROVED** [review]
LGTM! Plus an unrelated suggestion.

File: src/views/Home/components/About/AboutBox/AboutBox.scss [hunk 2]
```diff
@@ -17,11 +18,20 @@
...
-        @include responsiveness.font($responsive: 4.1vw, $min: 35px, $max: 80px);
+        @include responsiveness.font($responsive: 3.8vw, $min: 35px, $max: 80px);
         @include responsiveness.screen(smallest, phone) {
-            @include responsiveness.font($responsive: 7.1vw, $min: 22px, $max: 40px);
+            @include responsiveness.font($responsive: 7.1vw, $min: 20px, $max: 40px);
+            text-align: center;
             padding: 0 26px 0 30px;
         }
+        @include responsiveness.screen(smallest) {
+            padding: 0 0 0 0px;
```
Comment (w-gao) [inline src/views/Home/components/About/AboutBox/AboutBox.scss]: Also, this is just `padding: 0`, right?

Comment (krishnakpandian) [reply src/views/Home/components/About/AboutBox/AboutBox.scss]: Oh yeah this was probably me just copying from dev tools, I'll update this!

File: src/views/Home/components/About/AboutBox/AboutBox.scss (hunk 1 above)
Comment (krishnakpandian) [reply src/views/Home/components/About/AboutBox/AboutBox.scss:11]: Yeah I can add that as well, let me test and see how that'll look on smaller screens

**Comment by krishnakpandian** [comment]
I just made some edits, before I push, is this more in line in terms of fonts and background-colors? @w-gao 
verification-new-background

**Review by w-gao — APPROVED** [review]
Looks great to me! Thanks for the changes @krishnakpandian.

**Pull request merged**